server/src/builder.rs

```diff
@@ -135,7 +140,8 @@ pub(crate) fn build_offline_mode(
             client_features.clone(),
         )
     }
-    Ok((token_cache, features_cache, engine_cache))
+    // TODO: possibly need to resolve delta cache for offline mode?
```

Comment:
Personal feeling but I'd skip this one for now. I don't see much value in sending deltas when that event stream is frozen